src/app/screens/Settings.tsx

```diff
@@ -176,6 +176,7 @@ function Settings() {
                   });
                 }}
               >
+                <option value="alby">Alby</option>
```

Comment:
Should we let people know what service we use in the background? i.e. 

> Alby (via Coindesk)

Or... Should all options be implemented via our proxy and the setting will someone let proxy know which service should be used?

Just wondering what users care about here.